api-inference-community/tests/test_dockers.py

```diff
@@ -79,6 +79,12 @@ def test_espnet(self):
             "espnet/kamo-naoyuki_mini_an4_asr_train_raw_bpe_valid.acc.best",
         )
 
+    def test_fairseq(self):
```

Comment:
Adding a docker test (Checking that launching the docker works and most use cases show correct errors + cache is working.